Assets/Scripts/Bullet.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+
+public class Bullet : MonoBehaviour
+{
+    [SerializeField] 
+    private float timeToDestroy;
+    
+    [SerializeField] 
+    private float speed;
+    
+    private float timer;
+
+    private void Update()
+    {
+        if (timer > timeToDestroy)
+        {
+            Destroy(this);
+        }
+        
+        transform.Translate(Vector2.up * speed * Time.deltaTime);
+
+        timer += Time.deltaTime;
+    }
+    
+    private void OnTriggerEnter2D(Collider2D other)
```

Comment:
Czy 'other' to na pewno dobra nazwa? Może coś w stylu 'collider' będzie bardziej przejrzyste?